pkg/authenticator/config/config.go

```diff
@@ -28,91 +29,94 @@ const (
 	DefaultClientCertPath = "/etc/conjur/ssl/client.pem"
```

Comment:
Some linters are going to complain about `DefaultClientCertPath` not having a comment for an exported variable. Same with the other Default* consts below. You could satisfy linter for all of these with a comment before the `const (`, e.g. :
```
// Default settings (this comment added to satisfy linter)
const (
```